dlt-database/entity/0001-init_db/User.ts

```diff
@@ -26,7 +18,7 @@ export class User extends BaseEntity {
   @Column({ name: 'derive1_pubkey', type: 'binary', length: 32, unique: true })
   derive1Pubkey: Buffer
 
-  @CreateDateColumn({
+  @Column({
```

Comment:
If the `createdAt` column will only contain the timestamp of the write access than the annotation `@CreateDateColumn` will be prefered.
But keep in mind:
the `@CreateDateColumn` and/or `@UpdateDateColumn` will not treat the ms-dimension properly